### DISCUSSION TITLE
(goodbyedpi) Request for migration

### DISCUSSION THREAD

**Discussion by PRESFIL** [chocolatey-community/chocolatey-packages#2541] [Migrations]
### Why should the package be migrated to this Chocolatey Community Chocolatey Packages Repository?

I would like an automatic update of package definition.

### What is the link to the package hosted on Chocolatey Community Repository?

https://community.chocolatey.org/packages/goodbyedpi

### What is the link to the package source?

https://github.com/galeksandrp/chocolatey-packages/tree/goodbyedpi

### Responsibilities

- [X] I will be responsible for creating a PR to migrate the package.
- [X] I will be available to fix any issues with the package or the automatic updater!

**Comment by pauby** [comment]
As is mentioned over on the [README](https://github.com/chocolatey-community/chocolatey-packages/blob/master/README.md):

> NOTE: Currently we are unlikely to accept any new or migrated packages into this repository. Please raise a [Discussion](https://github.com/chocolatey-community/chocolatey-packages/blob/master/discussions) if you have a strong case for new or migrated packages.

I don't believe this is a strong case for the community to maintain it. If you need to have the package automatically updated, then please follow the [package triage](https://docs.chocolatey.org/en-us/community-repository/users/package-triage-process/) process.

**Reply by PRESFIL** [reply]
Ok, I have created [PR in original package](https://github.com/galeksandrp/chocolatey-packages/pull/5) repository.
Waiting for a response as stated in the package triage process.
